server/paymentProviders/stripe/creditcard.js

```diff
@@ -203,7 +206,11 @@ const createChargeAndTransactions = async (hostStripeAccount, { order, hostStrip
     hostFeeSharePercent,
   };
 
-  const platformFeeInHostCurrency = isSharedRevenue ? platformTip || 0 : fees.applicationFee;
+  const fxRate = convertFromStripeAmount(balanceTransaction.currency, balanceTransaction.amount) / order.totalAmount;
```

Comment:
Why not name it `hostCurrencyFxRate` ?

Reply:
What about computing the result of `convertFromStripeAmount(balanceTransaction.currency, balanceTransaction.amount)` before, storing it in a variable and naming it `amountInHostCurrency`?

Reply:
Thanks. 👍🏽 I've corrected those. 😄